src/components/OptionRow.js

```diff
@@ -174,6 +174,10 @@ class OptionRow extends Component {
                                 this.props.isDisabled && styles.cursorDisabled,
                                 this.props.shouldHaveOptionSeparator && styles.borderTop,
                             ]}
+                            accessibilityLabel={this.props.option.text}
+                            accessibilityRole="button"
+                            hoverDimmingValue={1}
+                            pressDimmingValue={0.8}
```

Comment:
the current default value for press opacity is the same
https://github.com/Expensify/App/blob/79ddcab5c118b2b5c3769150d3177b60bb58e86c/src/styles/variables.js#L143